less/admin.less

```diff
@@ -61,11 +61,96 @@
 }
 .StatisticsWidget-chart {
   clear: left;
-  max-width: 600px;
   margin: -20px -10px;
+}
+.StatisticsWidget-chart .chart-container {
+  position: relative;
+  .frappe-chart {
+    height: 160px !important;
+
+    .line-chart {
+      .dataset-0 {
+        opacity: 0.2;
 
+        path {
+          stroke: #7f7f7f !important;
+        }
+
+        circle {
+          fill: #7f7f7f !important;
+        }
+      }
+    }
+  }
+  .chart-legend {
+    display: none;
+  }
   // Hide the "last period" data from the tooltip
-  .chart-container .graph-svg-tip ul.data-point-list > li:first-child {
+  .graph-svg-tip ul.data-point-list > li:first-child {
     display: none;
   }
 }
+
+/*!
+ * Frappe Charts 1.0.0 by @frappe - https://frappe.io/charts
+ * License - MIT https://github.com/frappe/charts/blob/master/LICENSE
+ */
```

Comment:
So is this part taken from the Frappe sources? Why didn't we have this before and why is it needed?

Reply:
Yep, from the Frappe sources. It's needed for the data that appears when one hovers over a point on the graph, also known as the "comparison" view.
We didn't have it I assume because the extension wasn't finished.

With this CSS:
![image](https://user-images.githubusercontent.com/6401250/47817530-3dcdd800-dd2c-11e8-9937-cc4111e7f156.png)

Without:
![screen recording 2018-10-31 at 4 46 26 pm](https://user-images.githubusercontent.com/6401250/47817699-bc2a7a00-dd2c-11e8-837f-317835c2b702.gif)